### ISSUE TITLE
Too slow md-chips readonly

### ISSUE THREAD

**Issue by marvelsrp** [angular/material#11753]
#### Detailed Reproduction Steps: 

1. Create list with 100 items and 10 chips
```
this.items = Array.from(Array(100), (item, index) => ({
      name: `Item ${index}`,
      chips: Array.from(Array(10), (item, index) => `Chip ${index}`);
     })
 );

```
2. Render them items 
```
div(ng-repeat="item in controller.items")
    b {{item.name}}
    md-chips(ng-model="item.chips" readonly="true")
```
3. 

### What is the expected behavior?
Render immediate

### What is the current behavior?
Was added ~12000 watchers and make ~1000 digests (once per added chip).  Page will be lags 3-6 sec.

Digests were called from here: https://github.com/angular/material/blob/97455f529fd4bbf162d03f74e887dc6aa3f04a00/src/components/chips/js/chipDirective.js#L60

I think in readonly mode `$timout` with call `focusLastChipThenInput` can not be run.

**Comment by Splaktar** [comment]
Yeah, that seems like a performance problem. Can you please provide a CodePen demo?

**Comment by marvelsrp** [comment]
> Yeah, that seems like a performance problem. Can you please provide a CodePen demo?

https://codepen.io/marvelsrp/pen/RzZXgW

**Comment by Eduardo-Julio** [comment]
Currently experiencing the same issue
